docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3'
+
+services:
+  web:
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./web:/app
+    environment:
+      - MONGO_DB=mongodb://db:27017
```

Comment:
The part after `mongodb://` should be the name of the mongodb container, so in this case it should be `mongodb://mongodb:27017`

Reply:
Ah I see, thanks